notebooks/tutorials/hello-syft/01-hello-syft.ipynb

```diff
@@ -11,6 +11,45 @@
     "# Hello Syft"
```

Comment:
<p>no need to plot, just compute the mean and thats it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>